h2o-r/h2o-package/R/models.R

```diff
@@ -3131,14 +3132,16 @@ setClass(
     left_children = "integer",
     right_children = "integer",
     descriptions = "character",
-    model_id = "character",
+    model = "H2OModel",
```

Comment:
I like the previous version better because it was independent of a model and the Tree could have been potentially created using MOJO

Reply:
OH, this is indeed not needed in fact ! Great catch.